### ISSUE TITLE
Hitting exceeded quota for sec groups and secgroup-rules

### ISSUE THREAD

**Issue by guilhermesteinmuller** [vexxhost/atmosphere#467]
Since amphoras are created under admin project and quotas are the openstack defaults, we are hitting quota exceeded for secgroups and secgroups-rules

```
neutronclient.common.exceptions.OverQuotaClient: Quota exceeded for resources: ['security_group'].
```

**Comment by mnaser** [comment]
As a workaround, the quota for the security groups and security group rules should be set to unlimited in the admin project oworkaround.

**Comment by ricolin** [comment]
@guilhermesteinmuller  I notice security group already set to -1 for admin group in v1.1.0
https://github.com/vexxhost/atmosphere/commit/e7d6624bda0d2d3c11b44c8b867185f9aae0bd4d
I think this issue will be resolved after bump version to >= v1.1.0

**Comment by mnaser** [comment]
Good catch @ricolin -- I believe in this case indeed it was a deployment that was not updated.
